config/manager/manager.yaml

```diff
@@ -29,6 +30,7 @@ spec:
           - "--auditInterval=30"
           - "--port=8443"
           - "--logtostderr"
+          - "--exempt-namespaces=gatekeeper-system"
```

Comment:
we should introduce this new flag in all the yamls _after_ we cut a new release

Reply:
+1 I wonder if there is a way to streamline this process as we seem to have run into this issue a few times lately. Pending the outcome of the second comment, I can move this into a separate PR.

Reply:
sounds like this issue can be resolved by adding a staging deployment yaml

Reply:
+1, I can work on this tomorrow.

Reply:
Pending pr: #430

Reply:
Switched to using staging manifests